src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -23,15 +23,16 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 
-<div class="card">
+{% set paginationNum = 8 %}
+<div class="card" id="orderProductsPanel">
   <div class="card-header">
     <h3 class="card-header-title">
-      {{ 'Products'|trans({}, 'Admin.Global') }} ({{ orderForViewing.products.products|length }})
+      {{ 'Products'|trans({}, 'Admin.Global') }} (<span id="orderProductsPanelCount">{{ orderForViewing.products.products|length }}</span>)
     </h3>
   </div>
 
   <div class="card-body">
-    <table class="table">
+    <table class="table" id="orderProductsTable" data-currency-precision="{{ orderCurrency.precision }}">
```

Comment:
Great, one last thing to improve here The precision stored in currency is more destined to display (for formatted price for example)
But the computation one is higher (and I believe lots of prices are computed on the frontend side) So instead you should use `ComputingPrecision` (in the controller) to compute this value before injecting it in the page

Reply:
I fix it.